descriptions/rep/en_us.lua

```diff
@@ -220,7 +220,7 @@ local repCollectibles={
 	[581] = {"581", "Psy Fly", "Fly orbital familiar#Chases and deflects projectiles#Deals contact damage"}, -- Psy Fly
 	[582] = {"582", "Wavy Cap", "↑ +0.75 Fire Rate up#↓ -0.03 Speed down#Distorts the screen#Effect can be stacked"}, -- Wavy Cap
 	[583] = {"583", "Rocket in a Jar", "+5 bombs#If shooting, placed bombs turn into rockets that fire in the shooting direction"}, -- Rocket in a Jar
-	[584] = {"584", "Book of Virtues", "Spawns an orbital Wisp familiar that fires spectral tears but can be destroyed#Can be combined with a second active Item to create themed wisps#Turns first Devil Room into an Angel Room"}, -- Book of Virtues
+	[584] = {"584", "Book of Virtues", "Spawns an orbital wisp familiar that fires spectral tears but can be destroyed#Can be combined with a second active item to create themed wisps#Turns first Devil Room into an Angel Room, increases Angel Room chance, and allows Angel Room spawns even after Devil deals"}, -- Book of Virtues
```

Comment:
```suggestion
      [584] = {"584", "Book of Virtues", "Spawns an orbital wisp familiar that fires spectral tears but can be destroyed#Can be combined with a second active item to create themed wisps#Turns the first {{DevilRoom}}Devil Room into an {{AngelRoom}}Angel Room and increases the {{AngelRoom}}Angel Room chance#Allows both Devil and Angel deals to be taken"}, -- Book of Virtues
```
A small change, should sound a bit better.